demo/elastic/wazuh_cf_bootstrap_elastic.sh

```diff
@@ -18,7 +18,7 @@ master_ip=$(cat /tmp/wazuh_cf_settings | grep '^MasterIp:' | cut -d' ' -f2)
 worker_ip=$(cat /tmp/wazuh_cf_settings | grep '^WorkerIp:' | cut -d' ' -f2)
 kibana_ip=$(cat /tmp/wazuh_cf_settings | grep '^KibanaIp:' | cut -d' ' -f2)
 
-TAG="v3.13.0"
+TAG="v3.13.1"
```

Comment:
Why aren't we using this as a parameter?

Reply:
Missed that one. Thanks! Fixed on 19cccdadcbace719c7ce9c3cca7ba49c355230c1